processors/minsev/severity.go

```diff
@@ -0,0 +1,124 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package minsev // import "go.opentelemetry.io/contrib/processors/minsev"
+
+import (
+	"sync/atomic"
+
+	"go.opentelemetry.io/otel/log"
+)
+
+// Severity represents a log record severity (also known as log level). Smaller
+// numerical values correspond to less severe log records (such as debug
+// events), larger numerical values correspond to more severe log records (such
+// as errors and critical events).
+type Severity int
```

Comment:
What prevents us from using the severities defined in the API here? (and should we document it?)

Reply:
The API doesn't implement `Severitier`.

Reply:
I guess @dmathieu had in mind something like

```suggestion
type Severity log.Severity
```

Reply:
What difference does that make?

Reply:
I think it is a little clearer. But maybe @dmathieu had something else in mind.

Reply:
Maybe then we could get rid of `translations` an users could simple create severities e.g. like this?

```
s := minsev.Severity(log.SeverityInfo)
```

Reply:
> Maybe then we could get rid of `translations` an users could simple create severities like this?
> 
> ```
> s := minsev.Severity(log.SeverityInfo)
> ```

Is the idea that all the severity const are removed then? If so, this seems like a vague API and a poor design.

Instead of having users call:

```go
minsev.NewLogProcessor(p, minsev.SeverityInfo)
```

They would call:

```go
minsev.NewLogProcessor(p, minsev.Severity(log.SeverityInfo))
```

They have not syntactic hint about how to construct a `minsev.Severity`.

This would also mean we cannot default to `SeverityInfo` for zero values or things like:

```go
Severity(log.SeverityInfo).Severity().String()
```

will output `ERROR2`.

Reply:
Yeah, I meant it that it would be nicer to avoid duplicating every severity level. But you're right, doing so would make the API weird.